jscomp/core/lam_compile_primitive.ml

```diff
@@ -78,6 +117,32 @@ let translate loc (cxt : Lam_compile_context.t) (prim : Lam_primitive.t)
           | _ -> E.runtime_call Js_runtime_modules.option "nullable_to_opt" args
           )
       | _ -> assert false)
+  | Pimport -> (
+      match args with
+      | [ e ] -> (
+          match e.expression_desc with
+          | _ -> (
+              match output_prefix with
+              | Some output_prefix -> (
+                  let output_dir = Filename.dirname output_prefix in
+
+                  let module_id, module_value =
+                    match module_of_expression e.expression_desc with
+                    | [ module_ ] -> module_
+                    | _ -> assert false
```

Comment:
What's an example where this happens?

Reply:
One of example would be `let three = Js.import(3)`. It asserts false.